documentation/how-to/map-interaction.en.md

```diff
@@ -24,6 +24,16 @@ On double-tap or long-press on a layer, you get offered the following possibilit
 - *Reload data* to get the current data of a layer with remote sources.
 - *Show feature list* to show all the layer's features in the identification list.
 
+If you'd like to organize the feature list on the map canvas or in the 'Show feature list' in QField, you have the following options:
+
```

Comment:
```suggestion
## Sort layer features

If you'd like to configure the order of features in 'Show feature list' in QField, you have the following options:

```

This needs a header block.

What do you mean by "organize the feature list on the map canvas"?

Reply:
Agreed.

Reply:
My bad, you are right is only in the Show feature list of the layes.